static/js/components/ExternalLogins_test.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import React from "react"
+import { shallow } from "enzyme"
+import { assert } from "chai"
+
+import { TOUCHSTONE_URL } from "../lib/url"
+import ExternalLogins from "./ExternalLogins"
+
+describe("ExternalLogins component", () => {
+  const mountExternalLogins = (children, props = {}) =>
+    shallow(<ExternalLogins {...props}>{children}</ExternalLogins>)
```

Comment:
I don't think you need `children` here since `ExternalLogins` does nothing with it.